--- a/xmlplay_lib.js
+++ b/xmlplay_lib.js
@@ -636,7 +636,8 @@
         nt.ns.forEach (function (noot, i) { // speel accoord
             sLib.speel (t0 + i * ard, noot.inst, noot.mnum, noot.cnt, noot.dur, tf, nt.vce, noot.velo, noot.orn);
         });
-        putMarkLoc (nt); 
+        putMarkLoc (nt);
+        document.dispatchEvent(new CustomEvent('xmlplay-tempo', { detail: { tmp: nt.tmp } }));
         iSeq += 1;
     }
     var fout = t0 - gAccTime;   // echte tijd - ABC tijd
